content/en/community/journey.md

```diff
@@ -0,0 +1,35 @@
+# Join the Journey: Shape the Future Together!  
+
+## 🌱 **Community Member Journey: Grow & Contribute**  
+Whether you're new or experienced, our community helps you learn, engage, and thrive:  
+
+1. **Initial Interaction** – Discover us via our [forum](https://forum.communityhealthtoolkit.org/t/cht-calendar-stay-connected-with-regular-community-events/4641). You can see what is planned in the coming days and weeks, please use the link provided to subscribe to the CHT calendar.  
+2. **Onboarding** – Get access to demos, join our calls, and start exploring.  
+3. **Engagement** – Participate in forums, webinars, and roundup calls. Share ideas and feedback!  
+4. **Sustained Engagement** – Attend meetups, take [CHT Academy Courses](https://academy.communityhealthtoolkit.org/), and test new features.  
+5. **Thriving Contributor** – Lead as a contributor, governance member, or solution partner.  
+
+**Why join?** Influence the roadmap, connect with experts, and grow your impact in global health tech!  
+
+---
+
+## ⚡ **Squad Journey: Collaborate & Build**  
```

Comment:
There is a discrepancy between the squad stages on the [roadmap](https://github.com/medic/cht-docs/pull/1858) and the stages you added here. I suggest we merge them and keep them consistent so the community members won't get confused.